core-services/prow/02_config/openshift/oc-mirror/prerelease.yaml

```diff
@@ -0,0 +1,40 @@
+presubmits:
+- name: presubmit-test-pull
+  always_run: true
+  decorate: true
+  trigger: "(?m)bundle test( please)?"
+  rerun_command: "bundle test"
+  spec:
+    shareProcessNamespace: true
+    containers:
+    - name: main
+      image: quay.io/samwalke/gobuilder:latest
```

Comment:
@sfxworks would it be possible to use an inline Dockerfile build here or define the Dockerfile in `oc mirror` to build, so we don't have to use a registry outside of the CI registry for running jobs?

Reply:
This would take some time as inline Dockerfile builds are an OpenShift CI feature and these are native prow jobs. If you're willing to work around this preference for now I could at least include the dockerfile within the oc mirror PR while I do research on this. I actually forgot to include its contents when I was cleaning up the commits.

Reply:
I was looking for a way to push this image to the CI registry at least, but it was not clearly documented.

Reply:
Oh I see, their documentation on "Mirror Public Images" was there, but the link was broken on the homepage. I'll sift through this real quick and see what I can do.

Reply:
Oh I see, their document at https://docs.ci.openshift.org/docs/how-tos/external-images/ exists but the link was just broken on their homepage. I'll sift through this and see what I can do.

Reply:
@jpower432  After further research and talking to someone on the test platform team, it was recommended to just mirror the image if you _really_ desired the image ref to be within the openshift ci registry as opposed to quay with a dockerfile build attached to that. However I would need more time to accommodate the preferences laid out for this since the OpenShift CI operator is the only authorized service account that can push to the registry. This would also involve setting up OpenShift CI references. This is a lot of work and if you really would like me to push forward with this would not be complete in time by the date of the milestone set forth.

Reply:
Here's an example of how a periodic mirror would work as done by another repository. 
https://github.com/openshift/release/tree/master/core-services/image-mirroring/supplemental-ci-images/

Reply:
@sfxworks Thanks for looking into that. As long as everything is passing I think we can move forward with the config as-is and discuss another way forward with the images on a follow-on PR if need be.

Reply:
Sounds good. Yes all checks have been passed via last 5 commits to adjust for ci prow tests done on this PR.